conda-store/pyproject.toml

```
@@ -13,8 +13,10 @@ keywords = ["conda"]
 authors = [
   { name = "Christopher Ostrouchov", email = "chris.ostrouchov@gmail.com" },
 ]
-maintainers = [{ name = "Tania Allard", email = "trallard@bitsandchips.me" }]
-# TODO: update maintainers
+maintainers = [{ name = "Tania Allard", email = "trallard@bitsandchips.me" },
+    {name= "Pavithra Eswaramoorthy", email = " pavithraes@outlook.com"},
```

```suggestion
    {name= "Pavithra Eswaramoorthy", email = "pavithraes@outlook.com"},
```
(not sure if this matters)